scanapi/utils.py

```diff
@@ -1,7 +1,12 @@
 from scanapi.errors import InvalidKeyError, MissingMandatoryKeyError
 from scanapi.settings import settings
 
-ALLOWED_ATTRS_TO_HIDE = ("headers body").split()
+HEADERS = "headers"
```

Comment:
@gillianomenezes what do you think of moving all this hide related functions to a separated file dedicated for that?

Reply:
Good idea. I'll do that!